ansible.cfg

```diff
@@ -0,0 +1,6 @@
+[defaults]
```

Comment:
I figured it was reasonable to have most this defaulted, you can override with the cli args if you need to.